src/colecaoAutor.py

```diff
@@ -10,5 +10,5 @@ def incluir(self,autor: Autor):
         self.__lista.append(autor)
 
     @property
-    def num_de_autores(self):
-        return len(self.__lista)
\ No newline at end of file
+    def lista_de_autores(self):
+        return self.__lista[:]
```

Comment:
Opa, aqui você está retornando uma cópia da lista ou a própria lista? Por que se for a lista em si, estamos possibilitando qualquer pessoa em qualquer lugar remover itens da lista diretamente afetando todo o sistema. Quebra de encapsulamento.

Reply:
Fiquei com a mesma duvida ai consultei a documentação e vi que ele gera uma copia rasa.
Quem tem a minha copia não consegue alterar a minha lista, mas consegue acessar os originais que estão dentro da lista, no caso acessar os autores originais.
(Ele gera uma copia da lista que aponta para os mesmo objetos da original)
Eu poderia usar o método **deepcopy** para ele gerar uma copia dos elementos também.

Reply:
Acho que tudo bem fazer a cópia rasa, a lista continua protegida mas os elementos são os originais. Pode fazer merge :)